app/models/manageiq/providers/openstack/inventory/parser/network_manager.rb

```diff
@@ -79,7 +79,7 @@ def network_ports
       network_port = persister.network_ports.find_or_build(np.id)
       network_port.name = np.name.blank? ? mac_address : np.name
       network_port.status = np.status
-      network_port.admin_state_up = np.admin_state_up
+      network_port.admin_state_up = np.admin_state_up if np.admin_state_up.present?
```

Comment:
So even though there is a default_value at the db level if we explicitly set `admin_state_up` to `nil` it still sets it to `nil`

Reply:
Also, usually I'd `||` here but this is a boolean :smile: